docs: better semantic reading CONTRIBUTING.md

goobric opened the pull request:
Changed line 94, to give a fuller explanation of the reason to 'run tests'
Thank you Emma Dawson @emmalearnscode , for the wording/phrasing for line 94

<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->

## Fixes Issue

<!-- Remove this section if not applicable -->

<!-- Example: Closes #31 -->

## Changes proposed

<!-- List all the proposed changes in your PR -->

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
[x] - Correct; marked as done
[X] - Correct; marked as done

[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [ ] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

<!-- Add all the screenshots which support your changes -->

## Note to reviewers

<!-- Add notes to reviewers if applicable -->


<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1233"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

schmelto commented on CONTRIBUTING.md:
````diff
@@ -91,7 +91,7 @@ After making changes make sure that tests passes
 ```
 
-**2.** Run the tests by running this command
+**2.** <!--Run the tests by running this command--> The project runs automated tests when creating a PR. It tests components of the App. If you wish to run tests manually use the command ‘npm run cypress:run’ . If you need help to fix failing tests please comment on the PR asking for help.
````
```suggestion
**2.** The project runs automated tests when creating a PR. It tests components of the App. If you wish to run tests manually use the command ‘npm run cypress:run’ . If you need help to fix failing tests please comment on the PR asking for help.
```

schmelto reviewed (commented):
could we remove this comment?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

emmalearnscode reviewed (commented):
I'm wondering if instruction 1 is irrelevant in this section, especially if we make the text clearer like Mikal is suggesting. npm start doesn't really have anything to do with running tests and is not necessary to run the test command. Should we remove it? Otherwise we need to make it clearer what the purpose of doing instruction 1 is.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

goobric commented:
I have closed this PR. Thank you for your support.
For future reference, I can invite collaborators by following these instructions in this GH doc?
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository

goobric commented:
Do I now 'Delete branch' ?

emmalearnscode commented:
Since this is not resolved do you mind if I open a new issue asking someone else to replace the text @goobric ?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

goobric commented:
I selected 'Deleted Branch' and deleted it.
Now the message is asking "If you wish, you can delete this fork of EddieHubCommunity/LinkFree" in the settings!
I will delete the fork.
Very confused; as I am completely unaware if the proposed changes and have been agreed and implemented or not?!? :-/

emmalearnscode commented:
@goobric this line still needs to be added as it didn’t get merged: The project runs automated tests when creating a PR. It tests components of the App. If you wish to run tests manually use the command ‘npm run cypress:run’ . If you need help to fix failing tests please comment on the PR asking for help.
